README.md

```diff
@@ -212,10 +212,11 @@ scrape_configs:
+| basic-auth-password                 | REDIS_EXPORTER_BASIC_AUTH_PASSWORD               | Password for Basic Authentication with the redis exporter needs to be set together with basic-auth-username to be effective, conflicts with `basic-auth-hash-password`.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                       |
+| basic-auth-hash-password            | REDIS_EXPORTER_BASIC_AUTH_HASH_PASSWORD          | Bcrypt-hashed password for Basic Authentication with the redis exporter needs to be set together with basic-auth-username to be effective,  conflicts with `basic-auth-password`.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                            |
+| include-metrics-for-empty-databases | REDIS_EXPORTER_INCL_METRICS_FOR_EMPTY_DATABASES  | Whether to emit db metrics (like db_keys) for empty databases.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                        |
+| append-instance-role-label          | REDIS_APPEND_INSTANCE_ROLE_LABEL                 | Whether to append 'instance_role' label to all exporter metrics.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                       |
```

Comment:
Let's add 1-2 sentences here (or in a seperate section of the README) to explain what `instance_role` means and maybe even how to use it/think about it (you had a great explanation in that other thread, explaining that I can't rely on joining instance metrics).